### PR TITLE
Add NestJS tests and docs + Fastify bug fix

### PR DESCRIPTION
**Pull request by timokoessler**
- Adds end2end test for NestJS
- Add docs for NestJS
- Fix critical bug in Fastify wrapping

### CODE REVIEW

File: library/sources/fastify/wrapHandler.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import type { FastifyRequest } from "fastify";
 import { contextFromRequest } from "./contextFromRequest";
 
 export function wrapHandler(handler: Function): Function {
-  return (...args: unknown[]) => {
+  return function wrapped(...args: unknown[]) {
```
Comment (hansott) [inline library/sources/fastify/wrapHandler.ts:6]: Maybe we should use `arguments` in the rest of the function?

File: sample-apps/nestjs-fastify/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+## Stay in touch
+
+- Author - [Kamil Myśliwiec](https://twitter.com/kammysliwiec)
+- Website - [https://nestjs.com](https://nestjs.com/)
+- Twitter - [@nestframework](https://twitter.com/nestframework)
+
+## License
+
+Nest is [MIT licensed](https://github.com/nestjs/nest/blob/master/LICENSE).
```
Comment (hansott) [inline sample-apps/nestjs-fastify/README.md:99]: maybe we can remove some of these files? prettier, readme, eslint, ... we don't need it to run the app or mess with our code styling?

File: sample-apps/nestjs-fastify/test/app.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    await app.init();
+  });
+
+  it('/ (GET)', () => {
+    return request(app.getHttpServer())
+      .get('/')
+      .expect(200)
+      .expect('Hello World!');
+  });
+});
```
Comment (hansott) [inline sample-apps/nestjs-fastify/test/app.e2e-spec.ts:24]: we can remove the test dir maybe?

File: sample-apps/nestjs-fastify/README.md (hunk 2 above)
Comment (timokoessler) [reply sample-apps/nestjs-fastify/README.md:99]: Done

Comment (hansott) [reply sample-apps/nestjs-fastify/README.md:99]: Let's remove the readme too? or replace with content like our other sample apps?

Comment (timokoessler) [reply sample-apps/nestjs-fastify/README.md]: Forgot that, updated.

**Review by hansott — APPROVED** [review]

**Pull request merged**